include/libopencm3/stm32/common/flash_common_f234.h

```diff
@@ -50,7 +50,8 @@
 
 /* --- FLASH_ACR values ---------------------------------------------------- */
 
-#define FLASH_ACR_LATENCY_MASK		0x07
+#define FLASH_ACR_LATENCY_MASK		0x0f
+#define FLASH_ACR_LATENCY(w)		((w) & FLASH_ACR_LATENCY_MASK)
 #define FLASH_ACR_LATENCY_0WS		0x00
 #define FLASH_ACR_LATENCY_1WS		0x01
 #define FLASH_ACR_LATENCY_2WS		0x02
```

Comment:
I've been inclinded to just drop all of these.  latency is just a natural number.  as long as we have the mask we should be good.  doing it like this is probably good interim step, but in all reality, without versioned tags, there's no way of doing a timed deprecate,remove phase